docs/api-docs/storefront/graphql/graphql-faceted-textual-search.md

```diff
@@ -50,18 +50,18 @@ To use the Faceted and Textual Search feature, specify a filter in the argument
 ```
 These filters affect **both** the products and facets that are returned. For example, filtering by rating returns only products within the specified rating range and only facets that have products within the rating range.
 
-See [GraphQL Playground](/graphql-playground) for descriptions of each filter. 
+See the [GraphQL Storefront Playground](/graphql-playground) for descriptions of each filter. 
 
-## How to Get Products
+## Get products
 
 To get products, specify `products` as a field in `searchProducts`. Here is an example request that returns the first two products with a rating between three and five: 
 
 <!--
 type: tab
-title: GQL Query
+title: Example query
```

Comment:
If you do "Example query" and "Example response" for the tab titles, the tabs look a little repetitive (two "Example" right next to each other)